cmake/InstallLayout.cmake

```diff
@@ -54,8 +53,7 @@ function(install_symlink)
         install_symlink_impl(
           TO "../${rel_path}bin/${IN_FILE_NAME}"
           DESTINATION "usr/bin/${IN_LINK_NAME}"
-          COMPONENTS "${IN_COMPONENT}-el6"
-                     "${IN_COMPONENT}-el7"
+          COMPONENTS "${IN_COMPONENT}-el9"
```

Comment:
In 7.4 we drop support for el6 and el7?  (It is in there release notes I'm sure!)

Reply:
that's correct. 7.4 only supports RHEL 9.

Reply:
ack